var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -15,15 +15,18 @@ class RocmDebugAgent(CMakePackage):
 
```

Comment:
url = https://github.com/ROCm-Developer-Tools/rocr_debug_agent/archive/roc-3.7.0.tar.gz

Reply:
I don't think the URL has changed apart from the version number? Spack figures the new URL out itself -- i checked it works.